src/DependencyInjection/CompilerPass/CompatibleCheckPass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\DependencyInjection\CompilerPass;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class CompatibleCheckPass implements CompilerPassInterface
```

Comment:
I would rename this class to be more explicit about what it really does. Something like `DoctrineAttributesCheckPass`?

Reply:
For now it is just a Doctrine check, but i was thinking about using the pass for multiple compatible checks later. e.g. is symfony supporting attributes, or ApiPlatform and so on. Thats the reason i named it without a package as prefix.

Reply:
Pass name is changed to `DoctrineAttributesCheckPass` in the latest commit.